app/views/provider_interface/interviews/index.html.erb

```diff
@@ -12,6 +12,7 @@
         </h2>
 
         <% @interviews.each do |interview| %>
+          <a id="interview-<%=interview.id%>"></a>
           <div class="app-interviews__interview">
```

Comment:
To avoid having an empty link with the anchor, can we add the id to the existing div as so?
```
<div class="app-interviews__interview" id="interview-<%= interview.id %>">
```